quiz.js

```diff
@@ -0,0 +1,65 @@
+/*This is what the tree needs to look like*/
+//       *
+//      ***
+//     *****
+//    *******
+//   *********
+//  ***********
+// *************
+
+
+/*This section takes HTML fieldsets and turns them
+into variables*/
+
+var height = document.getElementById("height");
+var character = document.getElementById("character");
+var submit = document.getElementById("submit");
+function christmasTree () {
+};
+
+
+/*This section does both the event listeners */
+submit.addEventListener("click", treeGrowth); /* <-- Event Listener for clicking button*/
+height.addEventListener("keypress", keypress); /* <-- Event listener for pressing enter starts here*/
+character.addEventListener("keypress", keypress);
+
+function keypress(event) {
+    if (event.which === 13) { /* <-- check key code which is code that I got from the event list master when I consoled logged it*/
+        treeGrowth()
```

Comment:
Make sure you end each statement with a `;`. Later on in the course you will have a linter that will be monitoring your code, and it will scream at you each time you leave on out.